v1.0/update.md

```diff
@@ -8,16 +8,399 @@ The `UPDATE` [statement](sql-statements.html) updates rows in a table.
+|  1 | 10000.50 | michael  |
+|  2 |   4000.0 | jim      |
+|  3 |   8700.0 | pam      |
+|  4 |   3400.0 | andy     |
++----+----------+----------+
+(4 rows)
+~~~
+
+~~~ sql
+> UPDATE accounts SET balance = 5000.0 WHERE id = 2;
+
+> SELECT * FROM accounts;
+~~~
+~~~
++----+----------+----------+
+| id | balance  | customer |
++----+----------+----------+
+|  1 | 10000.50 | michael  |
+|  2 |   5000.0 | jim      |
+|  3 |   8700.0 | pam      |
+|  4 |   3400.0 | andy     |
++----+----------+----------+
+(4 rows)
+~~~
+
+~~~ sql
+> UPDATE accounts SET (balance, customer) = (9000.0, 'kelly') WHERE id = 2;
+
+> SELECT * FROM accounts;
+~~~
+~~~
++----+----------+----------+
+| id | balance  | customer |
++----+----------+----------+
+|  1 | 10000.50 | michael  |
+|  2 |   9000.0 | kelly   |
+|  3 |   8700.0 | pam      |
+|  4 |   3400.0 | andy     |
++----+----------+----------+
+(4 rows)
+~~~
+
+~~~ sql
+> UPDATE accounts SET balance = 6300.0, customer = 'stanley' WHERE id = 3;
+
+> SELECT * FROM accounts;
+~~~
+
+~~~
++----+----------+----------+
+| id | balance  | customer |
++----+----------+----------+
+|  1 | 10000.50 | michael  |
+|  2 |   9000.0 | kelly   |
+|  3 |   6300.0 | stanley  |
+|  4 |   3400.0 | andy     |
++----+----------+----------+
+(4 rows)
+~~~
+
+### Update with default values
+
+~~~ sql
+> UPDATE accounts SET balance = DEFAULT where customer = 'stanley';
+
+> SELECT * FROM accounts;
+~~~
+~~~
++----+----------+----------+
+| id | balance  | customer |
++----+----------+----------+
+|  1 | 10000.50 | michael  |
+|  2 |   9000.0 | kelly    |
+|  3 | NULL     | stanley  |
+|  4 |   3400.0 | andy     |
++----+----------+----------+
+(4 rows)
+~~~
+
+### Update from a SELECT statement
+~~~ sql
+> UPDATE accounts SET (balance, customer) = (SELECT balance, customer FROM accounts WHERE id = 2) WHERE id = 4;
+
+> SELECT * FROM accounts;
+~~~
+
+~~~
++----+----------+----------+
+| id | balance  | customer |
++----+----------+----------+
+|  1 | 10000.50 | michael  |
+|  2 |   9000.0 | kelly    |
+|  3 | NULL     | stanley  |
+|  4 |   9000.0 | kelly    |
++----+----------+----------+
+(4 rows)
+~~~
+
+### Update All Rows
+
+{{site.data.alerts.callout_danger}}If you don't use the WHERE clause to specify the rows to be updated, the values for all rows will be updated.{{site.data.alerts.end}}
+
+~~~ sql
+> UPDATE accounts SET balance = 5000.0;
+
+> SELECT * FROM accounts;
+~~~ 
+~~~
++----+---------+----------+
+| id | balance | customer |
++----+---------+----------+
+|  1 |  5000.0 | michael  |
+|  2 |  5000.0 | kelly    |
+|  3 |  5000.0 | stanley  |
+|  4 |  5000.0 | kelly    |
++----+---------+----------+
+(4 rows)
+~~~
+
+### Update and Return Values
+
+In this example, the `RETURNING` clause returns the `id` values of the rows updated, which are generated server-side by the `unique_rowid()` function. The language-specific versions assume that you have installed the relevant [client drivers](install-client-drivers.html).
```

Comment:
Some of the text taken from the `insert` example is not correct here. You need to remove the clause:

>which are generated server-side by the `unique_rowsid()` function.